sql/builtin_functions.cc

```diff
@@ -3058,11 +3058,12 @@ BoundFunction tryFct(const std::vector<BoundSqlExpression> & args)
         {
             ExcAssertEqual(boundArgs.size(), 1);
             try {
-                return storage = std::move(boundArgs[0](row, GET_LATEST));
+                return storage = boundArgs[0](row, GET_LATEST);
             }
             catch (const std::exception & exc) {
-                return storage = std::move(ExpressionValue(
-                    ML::getExceptionString(), Date::negativeInfinity()));
+                return storage
+                = ExpressionValue(ML::getExceptionString(),
```

Comment:
Disagree.  It's much easier to read with operators as the first thing on the line.